client/src/pages/CreateSession.js

```diff
@@ -22,12 +23,18 @@ const InputContainer = styled.div`
 flex-direction: row;
 `;
 
+const SERVER_URL = "http://localhost:3001"
+
 function CreateSession({ ...props }) {
     const history = useHistory();
 
     // TODO: Ask server for a session and navigate to custom session ??? unscoped
-    const generateSession = () => {
-        history.push('/order-screen')
+    const generateSession = async () => {
+      const { data: sessionId } = await axios.post(`${SERVER_URL}/session`, {
+        sessionName: document.getElementById("input-session-name").value,
```

Comment:
do we wanna be using `document.getElementById` instead using the input box and input vaue?

Reply:
I was lazy and tbh shouldn't have implemented the input thing in the first place. I forgot I could just use id. I think using id is better in this case cause I don't need to display it and have it constantly update.